tests/python_package_test/test_basic.py

```diff
@@ -473,7 +473,8 @@ def test_choose_param_value():
         "local_listen_port": 1234,
         "port": 2222,
         "metric": "auc",
-        "num_trees": 81
+        "num_trees": 81,
+        "n_iter": 10,
```

Comment:
```suggestion
        "n_iter": 13,
```

Could you please use a "weird" value, like `13`? In this test, I intentionally chose values that were not the current default or likely to be defaults in the future, to be sure the test doesn't accidentally pass if the default value is used instead of values from `params`.